tools/metrics.py

```diff
@@ -145,17 +139,21 @@ def main(releases=None):
 
                 if update.status == UpdateStatus.stable:
                     if not stablekarma_done:
-                        if comment.text == 'This update has reached the stable karma threshold and will be pushed to the stable updates repository':
+                        if comment.text == (
+                                'This update has reached the stable karma threshold and will be '
+                                'pushed to the stable updates repository'):
                             data['num_stablekarma'] += 1
                             stablekarma_done = True
-                        elif comment.text and comment.text.endswith('days in testing and can be pushed to stable now if the maintainer wishes'):
+                        elif comment.text and comment.text.endswith(
+                                'days in testing and can be pushed to stable now if the maintainer '
+                                'wishes'):
                             data['num_testingtime'] += 1
                             stablekarma_done = True
 
                 # For figuring out if an update has received feedback or not
                 if not feedback_done:
-                    if (not comment.user.name == 'bodhi'
-                        and comment.karma != 0 and not comment.anonymous):
+                    if (not comment.user.name == 'bodhi' and
+                            comment.karma != 0 and not comment.anonymous):
```

Comment:
@pypingou My vim linter (which is probably just flake8) complains if I put the and operator on the next line, so I think I'll keep it here. I don't personally have a preference, but I like my vim linter to be happy ☺